Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM ubuntu:16.04 
+USER root
+
+WORKDIR /tests
+
+ARG STAGE_CONFIG="{'zut':'allures'}"
+
+ENV MOZ_HEADLESS="1"
+ENV RUST_BACKTRACE="full"
```

Comment:
Is this [or should this be] configurable? I assume this is the reason for the vvvery vvverbose output when I run against stage.

Reply:
yea, let's drop this